devolutions-gateway/src/api/net.rs

```diff
@@ -44,6 +44,12 @@ pub async fn handle_network_scan(
             tokio::select! {
                 result = stream.recv() => {
                     let Some(entry) = result else {
+                        let _ = websocket
+                            .send(Message::Close(Some(axum::extract::ws::CloseFrame {
+                                code: axum::extract::ws::close_code::NORMAL,
+                                reason: std::borrow::Cow::from("network scan finished successfully"),
```

Comment:
This is nice, but I don’t think we ever use or even print the reason message on client side. So I would rather keep a straightforward code flow.